### PR TITLE
Added custom reporting documentation

### PR DESCRIPTION
**Pull request by kunagpal**
Here, the `teamcity` reporter for newman has been used as an example. Inspired by https://github.com/postmanlabs/newman/issues/910, related to https://github.com/leafle/newman-reporter-teamcity/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,6 +228,27 @@ Newman can output a summary of the collection run to a JUnit compatible XML file
...
 
+##### Creating and using custom reporters
+Newman also supports custom reporters, provided that the reporter works with Newman's event sequence. Working examples on how Newman reporters work can be found in [lib/reporters](https://github.com/postmanlabs/newman/tree/develop/lib/reporters). For instance, to use the [Newman teamcity reporter](https://github.com/leafle/newman-reporter-teamcity):
+
+- Add and install the reporter package. Note that the name of the package is of the form `newman-reporter-<name>`.<br/>
+```terminal
+npm install -S newman-reporter-teamcity
+```
+
+- Use the installed reporter, either via the CLI, or programattic usage. Here, the `newman-reporter` prefix is **not** required while specifying the reporter name in the options.<br/>
```
Comment (shamasis) [inline README.md]: Is programmatic spelt right?

**Comment by sergiorykov** [comment]
How can I get a clue what's wrong when reporter fails to load?
`npm install -S newman-reporter-teamcity`
`newman run .... -r teamcity`
`newman warning: "teamcity" reporter could not be loaded.`

At the same time from nodejs app in the same folder it works.

**Comment by kunagpal** [comment]
@sergiorykov It appears that your `newman` installation is global, while your `newman-reporter-teamcity` is installed locally. Check the result of `where newman`(Windows)/`which newman`(Linux and OSX). If the returned path is not in your project directory, you'll have to install the reporter globally as well: `npm i -g newman-reporter-teamcity`.

**Comment by sergiorykov** [comment]
thanks! It works. 

It would be nice to specify in documentation for custom reporter it. In awaiting commit specified local installation, while in 'Getting started' newman is installed globally. 

I use nodejs for utility projects and it's difficult to figure out why your internal loader can't find reporter - may be it's good to add this clarification in error message too.

File: README.md (hunk 1 above)
Comment (bondarewicz) [inline README.md]: programattic -> programmatic

**Comment by sergiorykov** [comment]
Thanks! 

I really appreciate how fast you respond and improve!

**Pull request merged**